test/test_plugins.py

```diff
@@ -266,8 +222,12 @@ def import_task_created_event(self, session, task):
         self.assertEqual(
             logs,
             [
-                "Singleton: {}".format(displayable_path(self.file_paths[0])),
-                "Singleton: {}".format(displayable_path(self.file_paths[1])),
+                "Singleton: {}".format(
+                    displayable_path(self.import_media[0].path)
+                ),
```

Comment:
This definitely looks better than using an f-string, but I can't wait for that to change with `pathlib` integration.

Reply:
> I can't wait for that to change with `pathlib` integration.

Oh yes, completely. This will remove amounts of boilerplate from `beets.util`.